src/overlay/test/OverlayTests.cpp

```diff
@@ -308,13 +315,14 @@ TEST_CASE("flow control byte capacity", "[overlay][flowcontrol]")
                     InitialSorobanNetworkConfig::FEE_TRANSACTION_SIZE_1KB;
                 configEntry.contractBandwidth().txMaxSizeBytes = maxTxSize;
                 configEntry.contractBandwidth().ledgerMaxTxsSizeBytes =
-                    InitialSorobanNetworkConfig::
-                        LEDGER_MAX_TRANSACTION_SIZES_BYTES;
+                    maxTxSize * 10;
                 res = txtest::makeConfigUpgradeSet(ltx, configUpgradeSet);
                 ltx.commit();
             }
             txtest::executeUpgrade(*app, txtest::makeConfigUpgrade(*res));
         };
+        upgradeApp(app1, txSize);
```

Comment:
why is this needed if the config is overridden on line 163?

Reply:
Probably due to ledger-wide limit... Also this section concerns upgrades, so it would make sense to do the upgrade for the setup.

Reply:
this section does upgrades in a controlled way, where order and delay between upgrades matters (and upgrades trigger overlay events as well). Why not just set ledger-wide limit in the same lambda you're using above?

Reply:
The config overrides just override the config in-memory. This is a test util sufficient for most of the tests that don't care about the ledger state. As upgrades in fact do care about ledger state, I feel that it's more appropriate to prepare the proper ledger state first via the proper (upgrade) mechanism.